### PR TITLE
U control chart calculator added 

### LINKED ISSUE #5329: Add U control chart calculator
## **Is your feature request related to a problem? Please describe.**

I would like to add U control chart under the statistics section

## **Describe the solution you'd like**

- A clear and concise description of what you want to happen.

### PR DESCRIPTION
**Pull request by Khushi-agarwal**
## Related Issue

Fixes #5329 
#### Describe the changes you've made

Added the calculator along with steps.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: appdata/menudata/statistics.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+</style>
+<div class="back" onclick="openOption('appdata/menu.html','Menu')"><img src="icons/back.png" alt=""></div>      
+
+<div class="menu">    
+    <div class="row">
+        <div class="col-md-3 col-6">
+            <div class="option">
+                <img id="ucontrolfav" src="icons/unfavourite.png" alt="" onclick="addtofavourites(this)">
+                
+                <span class="caltext" onclick="loadcalculator('study/uControlChart.html','U control chart Calculator')">U control chart Calculator</span>
```
Comment (sairish2001) [inline appdata/menudata/statistics.html:60]: You Put this file in `Calculators/` folder but here you put `study/.....` Have a look

**Review by sairish2001 — CHANGES_REQUESTED** [review]

File: appdata/menudata/statistics.html (hunk 1 above)
Comment (Khushi-agarwal) [reply appdata/menudata/statistics.html:60]: Oh!I will correct it.Should I put the file in study/... or write Calculators/.. there?

Comment (sairish2001) [reply appdata/menudata/statistics.html:60]: in Calculators/

**Comment by Khushi-agarwal** [comment]
@sairish2001 done the changes.

**Review by sairish2001 — APPROVED** [review]
Great Work :)

**Pull request merged**